android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -825,10 +828,13 @@ public void setAttributedStringText(AttributedStringProxy attrString)
 		Bundle bundleText =
 			AttributedStringProxy.toSpannableInBundle(attrString, TiApplication.getAppCurrentActivity());
 		if (bundleText.containsKey(TiC.PROPERTY_ATTRIBUTED_STRING)) {
+			//TIMOB-17210 Android: A textfield change listener is wrongly triggered also if the value is programmatically set before creation
+			disableChangeEvent = true;
```

Comment:
Can you change this code block's `disableChangeEvent ` handling to the following please?
```java
boolean wasDisabled = disableChangeEvent;
disableChangeEvent = true;
/* ... */
disableChangeEvent = wasDisabled;
```

The reason is because the `processProperties()` method already sets the `disableChangeEvent` to `true` and this method call will end up changing it back to `false` too soon. The above code will change it back to its original state. Thanks!

Reply:
You're right! I was still thinking why it's using wasDiabaled. Thanks for the explanation :)